### DISCUSSION TITLE
(npm) - react-json-view

### DISCUSSION THREAD

**Discussion by kaitoukid-1412** [facebook/docusaurus#8084] [Q&A] [answered]
Hi,

Im tryng build [react-json-view](https://www.npmjs.com/package/react-json-view) with my `example.mdx` post, but i cant 😔

### 
![image](https://user-images.githubusercontent.com/15698245/189571620-f3d732a0-d714-4066-b311-2d7c6216e94e.png)
![image](https://user-images.githubusercontent.com/15698245/189571724-89e1e250-189f-439d-b9dc-1590ba17db06.png)

### `<BrowserOnly/>`
![image](https://user-images.githubusercontent.com/15698245/189571882-914e5a37-60dc-4279-8ece-e454f7132248.png)
![image](https://user-images.githubusercontent.com/15698245/189571970-7d628078-a8eb-4f92-a1c0-eb94e99d77a9.png)

### `ExecutionEnvironment`
![image](https://user-images.githubusercontent.com/15698245/189572398-7c7dd237-a4fe-4227-945a-bfe8423fa384.png)
![image](https://user-images.githubusercontent.com/15698245/189572488-a6690c79-7c41-4ecb-8a85-14c59544a3d4.png)

### `siteConfig`

```js
import useDocusaurusContext from "@docusaurus/useDocusaurusContext";

const { siteConfig } = useDocusaurusContext();
```

- Docusaurus version: 2.1.0
- Node version: v16.17.0

I need help!

**Comment by Josh-Cena** [comment, marked as answer]
See 

https://github.com/facebook/docusaurus/blob/0a87116a1ce9be4dd2370c3edc0cf4215d6ff9d3/packages/docusaurus-plugin-debug/src/theme/DebugJsonView/index.tsx#L16-L29

https://github.com/mac-s-g/react-json-view/issues/121#issuecomment-670431408
